site/content/getting-started/_index.md

```diff
@@ -120,41 +96,12 @@ You should see the following:
 
 Congratulations, you have installed Contour and Envoy! Let's install a web application workload and get some traffic flowing to the backend.
  
-## 3. Install a kuard workload (simple application)  
-This section installs [kuard][9] to verify web traffic is flowing to the pods through envoy.
-
-Note: It is not recommended to expose kuard to the public.
-
-Install kuard by running the following:  
-
-```yaml
-$ kubectl apply -f https://projectcontour.io/examples/kuard.yaml
-```  
-Verify the pods and service are ready by running:
-```yaml
-$ kubectl get po,svc,ing -l app=kuard
-```  
-You should see the following:
-- 3 instances of pods/kuard, each with status **Running** and 1/1 **Ready**
-- 1 service/kuard CLUSTER-IP on port 80
-- 1 ingress on port 80
-
-
-Verify web access by browsing to [http://127.0.0.1.](http://127.0.0.1.) You can refresh multiple times to cycle through each pod workload.  
- 
-Congratulations, you have installed Contour with a backend web application on a kubernetes cluster! This installation has created the following:
+### Test it out!
 
-- Namespace <code>projectcontour</code>
-- Two instances of Contour in the namespace
-- A Kubernetes Daemonset running Envoy on each node in the cluster listening on host ports 80/443
-- A Service of <code>type: LoadBalancer</code> that points to the Contour’s Envoy instances
+Next, install a sample application to verify the quickstart installation in the [kuard workload section](/#install-a-kuard-workload-simple-application).
 
-Note: When you are done with the cluster, you can delete it by running:  
-```yaml
-$ kind delete cluster
-```  
 ---
-# Quickstart using Helm
+## Quickstart using Helm
```

Comment:
WDYT about moving the Helm-based install steps under "Install Contour and Envoy", above. So we'd have something like:

# Quickstart 
...
## Install Contour and Envoy
### Option 1: YAML
### Option 2: Helm
## Install and test a sample workload (kuard)

Could eliminate some duplication between the sections and might flow a little better.

I'd love to get support for tab panels on our site, so YAML vs Helm could be different tabs within a single table (this could be useful elsewhere for Ingress vs HTTPProxy vs Gateway, etc) but that's probably a separate issue